docs/application/dotnet/guides/personal/account.md

```diff
@@ -463,22 +464,22 @@ The following table lists the account properties that can be modified.
-| User name           | `String`                           | Yes       | Identity of an account.If the display name and email ID are not set for an account, the user name is shown for the account on the Accounts screen in the Setting application. |
-| Display name        | `String`                           | No        | Display name of an account.Display name is shown for the account on the Accounts screen in the Setting application. |
-| Email ID            | `String`                           | No        | Email ID of an account.If the display name is not set for an account, the email ID is shown for the account on the Accounts screen in the Setting application. |
-| Package name        | `String`                           | No        | One of an account package IDs, like the app ID.If the package name is not set for an account, the app ID is used as a package name. |
-| Icon path           | `String`                           | No        | Icon path of an account.The icon is shown through the registered icon path as an account icon on the Accounts screen in the Setting application. |
-| Domain name         | `String`                           | No        | Domain name of an account.               |
-| Access token        | `String`                           | No        | Access token of an account.              |
-| Auth type           | `Integer`                          | No        | Authentication type, such as oauth or xauth. |
-| Capability          | Key-value `string`-`integer` pairs | No        | Capability of an account.                |
-| Secret              | `Integer`                          | No        | The secret value is used to decide whether the account is shown on the Accounts screen in the Setting application. |
-| Sync support        | `Integer`                          | No        | Current synchronization status.          |
-| Source              | `String`                           | No        | Source of an account.                    |
-| User text           | `String`                           | No        | String array, which you can use freely.  |
-| User int            | `Integer`                          | No        | Integer array, which you can use freely. |
-| Custom              | Key-value `string` pairs           | No        | Key-value pairs, which you can use freely. |
-| Safe account handle | `SafeAccountHandle`                | No        | Read-only account handle of an account.  |
+| User name           | String                           | Yes       | Identity of an account.<br>If the display name and email ID are not set for an account, the user name is shown for the account on the Accounts screen in the Setting application. |
+| Display name        | String                           | No        | Display name of an account.Display name is shown for the account on the Accounts screen in the Setting application. |
```

Comment:
line 468: The separated lines are merged as one line.
line 495: The separated lines are merged as one line.
(Email capability:, Photo capability:, Video capability:, Music capability:, Document capability:, Message capability:, Game capability:)